.gitignore

```diff
@@ -23,6 +23,7 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+yarn.lock
```

Comment:
Nice work @johnkegz , however, why did you add `yarn.lock` to `.gitignore`

Reply:
I had put it there because I thought some people use npm which generates `package-lock.json` file but anyway let me remove it from there.